app/templates/email/ResetPassword.php

```diff
@@ -47,7 +47,7 @@ public static function setEmailBody($user, $host, $resetToken)
             <div style="padding:20px 10px;">
                 <p>Dear ' . $user->name . ',</p>
                 <p>Click on the link below to reset your password.</p>
-                <a href="http://' .$host. '/resetpassword?token=' . $resetToken . '">Reset Password</a>
+                <a href="http://' .$host. '/reset-password?token=' . $resetToken . '">Reset Password</a>
```

Comment:
Please remove http from the URL. The URL should be href="//'.$host.'/reset ......

Reply:
Changes incorporated!